homeassistant/components/unifi/config_flow.py

```diff
@@ -123,8 +123,9 @@ async def async_step_user(self, user_input=None):
 
                 return await self.async_step_site()
 
-        host = self.config.get(CONF_HOST)
-        if not host and await async_discover_unifi(self.hass):
+        if not (host := self.config.get(CONF_HOST, "")) and await async_discover_unifi(
```

Comment:
Are these changes here to increase test coverage or fix a bug? The previous code was easy to follow, and I'm also fairly new to the walrus operator: can you help me understand the benefit in this specific case? (I think i get the gist of why this operator is useful in other contexts)

Reply:
It is merely a simplification of assignment. This means you can evaluate and assign at the same time